backend/dataall/core/organizations/services/organization_service.py

```diff
@@ -260,3 +296,11 @@ def resolve_organization_by_env(uri):
         with context.db_engine.scoped_session() as session:
             env = EnvironmentRepository.get_environment_by_uri(session, uri)
             return OrganizationRepository.find_organization_by_uri(session, env.organizationUri)
+
+    @staticmethod
+    def list_group_organizaton_perissions(organizationUri, groupUri):
```

Comment:
We could protect the API call with `@ResourcePolicyService.has_resource_permission(GET_ORGANIZATION) ` otherwise anyone can check organization permissions of any orga


nit: typo permissions in name `list_group_organizaton_perissions`

Reply:
corrected